core/src/test/java/org/elasticsearch/common/geo/GeoDistanceTests.java

```diff
@@ -95,8 +98,8 @@ public void testWriteToSerializationBWC() throws Exception {
             geoDistance.writeTo(out);
             try (StreamInput in = out.bytes().streamInput()) {
                 in.setVersion(out.getVersion());
-                GeoDistance copy = GeoDistance.readFromStream(in);
```

Comment:
problem here is that this does not
test the readFromStream of nodes versioned <= 5.3.2